packages/flex-plugin-utils-http/src/http.ts

```diff
@@ -39,6 +40,8 @@ export interface HttpConfig extends OptionalHttpConfig {
   baseURL: string;
 }
 
+const propsToAdd = ['react', 'react-dom', 'flex-plugins-api-toolkit', '@twilio/flex-ui'];
```

Comment:
we need to rename them in the user agent to be `react_dom` (i.e. replace all `-` with `_`)

Reply:
it is!